paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -263,13 +277,16 @@
                                         }
                                     }
                                 }
-                            }
+                            },
+                            "required": [
+                                "target_average_value"
+                            ]
                         }
                     },
                     "patternProperties": {
-                        "^[a-z]([-a-z0-9]*[a-z0-9])?$": {
+                        "(?!(^cpu|memory|http|uwsgi)$)(^[a-z]([-a-z0-9]*[a-z0-9])?$)": {
```

Comment:
This regext looks pretty complex.
Can you say in words what this (should) represents?

Reply:
The first part exclude the keywords from patterned properties. The second half is copied from kubernetes documentation. That is metrics name.

Reply:
I'll add this to paasta docs once I finish. I can't add comments to this json (I'm happy about this somehow).